creator-node/src/logging.js

```diff
@@ -10,11 +10,13 @@ function RawStdOutWithLevelName() {
   return {
     write: (log) => {
       // duplicate log object before sending to stdout
-      const clonedLog = Object.assign({}, log)
+      const clonedLog = { ...log }
 
       // add new level (string) to level key
       clonedLog.logLevel = bunyan.nameFromLevel[clonedLog.level]
 
+      // stringify() uses the safeCycles() replacer, which returns '[Circular]'
+      // when circular references are detected
```

Comment:
@joaquincasares can we add a link to the bunyan safeCycles code eg https://github.com/trentm/node-bunyan/blob/master/lib/bunyan.js#L1155-L1190

Reply:
Updated! Thanks!